README.rst

```diff
@@ -242,6 +251,33 @@ in terms of disk space.
 
 Whether ping checks are created automatically for devices.
 
+``OPENWISP_MONITORING_AUTO_DEVICE_CONFIG_CHECK``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-------------+
+| **type**:    | ``bool``    |
++--------------+-------------+
+| **default**: | ``True``    |
++--------------+-------------+
+
+This setting allows you to choose whether `config_modified <#Configuration>` checks should be
```

Comment:
see how it looks: https://github.com/openwisp/openwisp-monitoring/blob/125f1c7fb11280cb3435f6545cf7171b893d9aaa/README.rst#id2

Always do these manual visual checks before publishing docs.